lib/install/web.rb

```diff
@@ -22,7 +22,7 @@ def apply_template!
 
     fail Rails::Generators::Error, message
   end
-  if options[:database] == "postgresql" && options[:skip_test]
+  if options[:database] == "postgresql" && options[:skip_test] && options[:skip_rubocop]
```

Comment:
I need to manually test against this branch to ensure the application template works.